host-controller/src/main/java/org/jboss/as/host/controller/ManagedServer.java

```diff
@@ -700,6 +711,25 @@ private static InternalState nextState(final InternalState state, final Internal
         return protocolClient.execute(listener, operation, OperationMessageHandler.DISCARD, OperationAttachments.EMPTY);
     }
 
+    private AsyncFuture<OperationResponse> shutdown(int timeoutInSeconds, final BlockingQueueOperationListener<TransactionalProtocolClient.Operation> listener) throws IOException {
+        final ModelNode operation = new ModelNode();
+        operation.get(OP).set(SHUTDOWN);
+        operation.get(OP_ADDR).setEmptyList();
+        operation.get(TIMEOUT).set(timeoutInSeconds);
+
+        return protocolClient.execute(listener, operation, OperationMessageHandler.DISCARD, OperationAttachments.EMPTY);
+    }
+
+    private AsyncFuture<OperationResponse> reload(int timeoutInSeconds, final BlockingQueueOperationListener<TransactionalProtocolClient.Operation> listener, int operationID) throws IOException {
+        final ModelNode operation = new ModelNode();
+        operation.get(OP).set(RELOAD);
+        operation.get(OP_ADDR).setEmptyList();
+        operation.get(SUSPEND_TIMEOUT).set(timeoutInSeconds);
+        operation.get("operation-id").set(operationID);
```

Comment:
The use of this "operation-id" has been clear up by @bstansberry in https://issues.jboss.org/browse/WFCORE-1791. The "operation-id" here makes sense and it is working fine. It should not be removed from reload op.